cookbook/basic_rag_example/local_embeddings.py

```diff
@@ -0,0 +1,60 @@
+"""A simple script for using RAG to ask questions about recent soccer results."""
+import os
+
+import numpy as np
+import openai
+import pandas as pd
+
+from mirascope import OpenAIChat, Prompt
+
+os.environ["OPENAI_API_KEY"] = "YOUR_API_KEY"
+
+soccer_info = [
+    "Manchester City won the league, continuing their domestic dominance.",
+    "In La Liga, FC Barcelona emerged victorious with a sturdy defense.",
+    "Dortmund slipped up in the last matchday to hand Bayern the Bundesliga title.",
+    "For the first time since Diego Maradona's days, Napoli lifted the Serie A trophy.",
+]
+
+client = openai.OpenAI()
+embeddings_model = "text-embedding-ada-002"
+df = pd.DataFrame(columns=["texts", "embeddings"])
```

Comment:
instead of concatenating dataframes, just create a python dictionary and then create the data frame e.g. `pd.DataFrame({"texts": texts, "embeddings": embeddings})`